core/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/http/appRoutesModels.scala

```diff
@@ -45,6 +47,10 @@ final case class ListAppResponse(googleProject: GoogleProject,
                                  appName: AppName,
                                  diskName: Option[DiskName])
 
+final case class BatchNodepoolCreateRequest(numNodepools: NumNodepools,
+                                            kubernetesRuntimeConfig: Option[KubernetesRuntimeConfig],
+                                            clusterName: Option[KubernetesClusterName])
```

Comment:
Im adding an optional cluster name to this request here, because it's impossible to verify cluster state with our strategy of asking GKE directly otherwise. Users/UIs will not typically specify this.

Reply:
Maybe add this to swagger as well?